src/page/courier/TaskListPage.js

```diff
@@ -46,16 +51,34 @@ class TaskListPage extends Component {
 
   taskList = null
 
+  static navigationOptions = ({navigation}) => {
+    const { params } = navigation.state
+    return {
+      headerRight: (
+        <View style={{flex: 1, flexDirection: 'row', alignItems: 'flex-end'}}>
+          <Button transparent onPress={() => navigation.state.params.toggleFilterModal()}>
+            <Icon name="funnel" />
```

Comment:
This icon seems weird for this purpose - did you see it somewhere else ? I would have suggested using https://fontawesome.com/icons/sliders-h?style=solid
Also icon color should be white.